module3/Projects/PairProjectCodeReview.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Peer Code Review
+---
+
+You will be paired with one person from _a different_ group for today's peer review session.
+
+1. Share a link to your github repository.
+2. Spend 20 minutes to review the other teams' code. Answer the following questions (write them down to share!)
+    * Where are they performing the conversion of PriceInCents to PriceInDollars?
+    * When editing an item, what kind of data is expected in the 'price' field? Are there any disadvantages to this datatype? Is there another place this conversion could be made?
+    * What differences are there in the migration files between their project and yours?
+    * How is the total spent by customer calculated?
+    * In reviewing the other group's code - what questions do you have for them, or for an instructor?
```

Comment:
Might be nice to also add a generic, "Was there anything the other group did that made their code easier to understand" and "Were there any sections of the code you found particularly difficult to understand?" or something similar to get students thinking about keeping an eye towards code readability and future developers who will need to work in their code.